integrations/philip-click-up/main.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any
+
+from port_ocean.context.ocean import ocean
+from enum import StrEnum
+from loguru import logger
+from click_up.client import ClickUpClient
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+
+class ObjectKind(StrEnum):
+    TEAM = "team"
+    PROJECT = "project"
+    ISSUE = "issue"
+
+
+# Required
+# Listen to the resync event of all the kinds specified in the mapping inside port.
+# Called each time with a different kind that should be returned from the source system.
+@ocean.on_resync(ObjectKind.TEAM)
+async def on_resync_team(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+
+    client = ClickUpClient(click_up_token=ocean.integration_config("clickup_api_key"))
+
+    async for teams in client.get_teams():
+        logger.info(f"Received {len(teams)} teams")
+
+        yield teams
```

Comment:
Teams is not really paginated so you might as well just return it using the return keyword instead of yield

`return await client.get_teams()`

then change the type from `ASYNC_GENERATOR_RESYNC_TYPE` to `RAW_RESULT`


Please apply it to the other resync events too

Reply:
Done.


I left the issues resync event as ASYNC_GENERATOR_RESYNC_TYPE since with the pagination I assume we would like all the issues at once, similar to the Jira Issue implementation.